src/app/shared/directives/mdl-textarea-autoresize.directive.ts

```diff
@@ -0,0 +1,28 @@
+import { AfterViewChecked, Directive, ElementRef, HostListener } from '@angular/core';
+
+
+@Directive({
+  // tslint:disable-next-line
+  selector: '[mdl-textarea-autoresize]'
+})
+export class MdlTextAreaAutoresizeDirective implements AfterViewChecked {
+  private textArea: HTMLTextAreaElement;
+
+  constructor(public elementRef: ElementRef) {}
+
+  @HostListener('input', ['$event.target'])
```

Comment:
Please add functionality to use "Enter" button while typing to move to a new line